packages/bazinga64/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "outDir": "dist",
-    "rootDir": "src/main"
+    "rootDir": "src"
```

Comment:
:+1:

Reply:
Were `src/main` came from: https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html 😃